src/components/venuelist/venuelist.js

```diff
@@ -21,37 +24,45 @@ export default class VenueList extends React.Component {
     let day = this.days[dayIndex];
     let dayOpen = day + "_open";
     let dayClose = day + "_close";
-
     if (this.props.venues) {
       return (
-        < ul className="venues-list" >
-          {
-            this.props.venues.map((venue, index) => {
-              {
-                index++;
-              }
-              return (
-                <li key={`marker-${index}`} className="venue">
-                  <img src={require("../../assets/pins/pin-" + index + ".svg")} />
-                  <div className="description">
-                    <strong>{venue.name}</strong>
-                    <p>
-                      {venue[dayOpen] ? (
-                        <span>
-                          {this.capitalize(day)}, open{" "}
-                          {venue[dayOpen].slice(0, -3)}, closes at{" "}
-                          {venue[dayClose].slice(0, -3)}
-                        </span>
-                      ) : (
-                          <span></span>
-                        )}
-                    </p>
+        <ul className="venues-list">
+          {this.props.venues.map((venue, index) => {
+            {
+              index++;
+            }
+            return (
+              <li key={`marker-${index}`} className="venue">
+                <img src={require("../../assets/pins/pin-" + index + ".svg")} />
+                <div className="venue-details">
+                  <h2>{venue.name}</h2>
+                  <p>
+                    {venue[dayOpen] ? (
+                      <span>
+                        {this.capitalize(day)}, open{" "}
+                        {venue[dayOpen].slice(0, -3)}, closes at{" "}
+                        {venue[dayClose].slice(0, -3)}
+                      </span>
+                    ) : (
+                      <span></span>
+                    )}
+                  </p>
+                  <div class="venue-icons">
+                    {venue.toilet ? <WC /> : <WC />}{" "}
+                    {venue.wheelchair_access ? <Access /> : <Access />}
```

Comment:
Same comment as above

Reply:
oh yeah, good spot, i put that in to test because none of the data had true values for these. will fix tonight.